.github/workflows/build_push-update_context_filter.yml

```diff
@@ -0,0 +1,54 @@
+name: "Build-push-update_context-filter"
+
+on:
+  push:
+    # branches:
+    #   - 'master'
+    tags:
+      - "context-filter/v*.*.*"
```

Comment:
Also needs to be underscore. This name will typically match the directory name of the module.